frontend/src/pages/Dashboard/index.tsx.show

```diff
@@ -0,0 +1,664 @@
+import React from 'react';
+import {
+  Button,
+  Table,
+  Card,
+  Modal,
+  Divider,
+  Input,
+  Radio,
+  Descriptions,
+  Tag,
+} from 'antd';
+import axios from 'axios';
+import './index.css';
+
+const { Search } = Input;
+
+/**
+ * @description
+ *
+ * @param {text}
+ */
+function loadAllModels() {
+  const targetUrl = 'http://155.69.146.35:8000/api/v1/model/';
+  axios
+    .get(targetUrl)
+    .then(function (response) {
+      // handle success
+      console.log(response);
+    })
+    .catch(function (error) {
+      // handle error
+      console.log(error);
+    })
+    .then(function () {
+      // always executed
+    });
+}
+
+function loadModelById(id) {
+  const targetUrl = 'http://155.69.146.35:8000/api/v1/model/';
```

Comment:
Hard code for now?

Reply:
url will remove after testing

Reply:
`index.tsx.show` is a backup file for demo screenshot, for web application please check `index.tsx` file.